src/converters/fromZigbee.ts

```diff
@@ -962,7 +962,18 @@ export const power_on_behavior: Fz.Converter = {
 };
 export const ias_no_alarm: Fz.Converter = {
     cluster: "ssIasZone",
-    type: ["attributeReport", "commandStatusChangeNotification"],
+    type: "commandStatusChangeNotification",
+    convert: (model, msg, publish, options, meta) => {
+        const zoneStatus = msg.data.zonestatus;
+        return {
+            tamper: (zoneStatus & (1 << 2)) > 0,
+            battery_low: (zoneStatus & (1 << 3)) > 0,
+        };
+    },
+};
+export const ias_no_alarm_report: Fz.Converter = {
+    cluster: "ssIasZone",
+    type: "attributeReport",
     convert: (model, msg, publish, options, meta) => {
         const zoneStatus = msg.data.zoneStatus;
```

Comment:
```suggestion
        const zoneStatus = msg.data.zoneStatus ?? msg.data.zonestatus;
```

Can you try with just this change?

Reply:
It works. Actually, I wondered if it was better to make 2 functions or just add a condition.
![Capture d’écran du 2025-07-08 19-55-18](https://github.com/user-attachments/assets/d271fc72-1e9f-4d95-9037-a0ed0d8d8ba5)